core/libp2p/protocol_muxer/multiselect/message_manager.cpp

```diff
@@ -0,0 +1,351 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "libp2p/protocol_muxer/multiselect/message_manager.hpp"
+
+#include <string_view>
+
+#include <boost/format.hpp>
+
+OUTCOME_CPP_DEFINE_CATEGORY(libp2p::protocol_muxer, MessageManager::ParseError,
+                            e) {
```

Comment:
use clang-format

Reply:
Already, the line is exactly 80 symbols long